### ISSUE TITLE
Privacy Manifest (PrivacyInfo.xcprivacy) is invalid in 6.10.2

### ISSUE THREAD

**Issue by Westacular** [twilio/twilio-voice-ios#125]
### Description

Apple has started issuing warnings related to privacy manifests when submitting apps to App Store Connect.

For an app submission that included the latest version (6.10.2) of the Twilio Voice iOS SDK, we got the warning:

> ITMS-91056: Invalid privacy manifest - The PrivacyInfo.xcprivacy file from the following path is invalid: “Frameworks/TwilioVoice.framework/PrivacyInfo.xcprivacy”. While no action is required at this time, starting May 1, 2024, when you upload a new app or app update, keys and values in your app’s privacy manifest must be in a valid format. For more details about privacy manifest files, visit: https://developer.apple.com/documentation/bundleresources/privacy_manifest_files.

I took a look at the PrivacyInfo.xcprivacy in TwilioVoice, and my only guess as to what is wrong is regarding

```
	<key>NSPrivacyAccessedAPITypes</key>
	<array>
		<dict/>
	</array>
```

where it's an empty dict inside the array; perhaps it should just be an empty array instead?

### Versions

All relevant version information for the issue.

#### Voice iOS SDK

6.10.2 via CocoaPods 1.15.2

#### Xcode

15.2

**Comment by dipankadas** [comment]
@Westacular Thanks for bringing it to our attention. We are looking at it internally and will have an update soon.

**Comment by mithleshgomotive** [comment]
+1 to above. Can we please plan this ?

**Comment by dipankadas** [comment]
We will be making a new release next week.

**Comment by dipankadas** [comment]
A new release had been made with version 6.10.3. This fixes the privacy manifest issue.

**Comment by iseebi** [comment]
I have linked 6.10.3 to my app via CocoaPods, but I think the definition of NSPrivacyAccessedAPICategorySystemBootTime is missing.
I checked with [app_store_required_privacy_manifest_analyser](https://github.com/crasowas/app_store_required_privacy_manifest_analyser) and it outputs that this definition is missing.

```
Analyzing TwilioVoice.framework ...
💡 Found privacy manifest file(s): 1
[0] /****/****.app/Frameworks/TwilioVoice.framework/PrivacyInfo.xcprivacy
API usage analysis result(s): 2
[0] NSPrivacyAccessedAPICategoryFileTimestamp:stat:/****/****.app/Frameworks/TwilioVoice.framework/TwilioVoice
[1] NSPrivacyAccessedAPICategorySystemBootTime:mach_absolute_time:/****/****.app/Frameworks/TwilioVoice.framework/TwilioVoice
🛠️  Descriptions for the following required API reason(s) may be missing: 1
[0] NSPrivacyAccessedAPICategorySystemBootTime
```

**Comment by bobiechen-twilio** [comment]
Hi @iseebi 

Thanks for again for reporting this. We have released the [Voice iOS SDK 6.10.4](https://www.twilio.com/docs/voice/sdks/ios/changelog#6104) to include the usage of this API in the privacy manifest.

**Comment by dipankadas** [comment]
This is fixed. Closing the issue.
